01_fib/saurabhvyas9/main.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func fib(n int) {
+	a, b := 0, 1
+	for i := 1; i <= n; i++ {
+		fmt.Fprintln(out, b)
+		c := a + b
```

Comment:
nitpick: this local var `c` isn't strictly necessary, consider skipping local assignments unless they're necessary for functionality or (rarely) readability

Reply:
Resolved in last commit

Reply:
Done.